chaincode-labs/chaincode-podcast/chaincode-decoded-blockchain.md

```diff
@@ -1,461 +1,387 @@
 It just gives you more lottery tickets.
+
+Mark Erhardt: 00:03:06
+
 There's often this misconception that there's a stack of work to power through, but that's not the case.
-Every single attempt at taking a block template and hashing it to check whether the hash is low enough to constitute a valid block is completely independent.
-So there is no progress here.
+Every single attempt - taking a block template and hashing it to check whether the hash is low enough to constitute a valid block - is completely independent.
+There is no progress here.
 If you've done 10 hashes already, you're no way ahead from somebody that's just doing their first hash.
-Basically, this is what makes it a lottery more and why not the fastest miner always wins.
-Every single attempt has the same likelihood of winning and you don't know whether any of the tickets are more likely to win until you try them.
+This is what makes it (more) a lottery, and not - the fastest miner always wins.
+Every single attempt has the same likelihood of winning, and you don't know whether any of the tickets are more likely to win until you try them.
 
-Speaker 1: 00:03:48
+Host: 00:03:48
 
-When you say fastest, you mean more hashes per second compared to another minor, like more other hardware that isn't computing as quickly?
+When you say fastest, you mean more hashes per second compared to another miner, like more other hardware that isn't computing as quickly?
 
-Speaker 0: 00:03:56
+Mark Erhardt: 00:03:56
 
 Yeah, exactly.
-Optimally, if you ignored latency and all that, basically this would lead to people having the relative chance of winning the next block as the ratio of hash rate they have of the global hash rate.
+Optimally, if you ignored latency and all that, basically this would lead to people having the relative chance of winning the next block as the ratio of hashrate they have of the global hashrate.
 So that's the optimum.
-So it's a little less exact that because there is latency and it takes a moment for people to validate the new block when it gets propagated and things like that.
-But it's pretty good.
-And given the long window, the long target block interval of, well, 10 minutes targeted, the latency is actually a very small portion of the overall window in which a new block is being tried to find.
-And interestingly enough, since there is no progress and all of these events are independent, however long it has been since the last block is being found, the next block is expected to take 10 minutes.
+It's a little less exact because there is latency, and it takes a moment for people to validate the new block when it gets propagated and things like that, but it's pretty good.
+Given the long window, the long target block interval of 10 minutes, the latency is actually a very small portion of the overall window in which a new block is tried to (be found).
```

Comment:
> Typically, parentheses are used to include additional context or information not explicitly mentioned, aiming to preserve the original intent while enhancing clarity. However, the introduction of parentheses in this case seems to substitute for what was omitted, rather than adding missing context. This raises a question: would it be more effective to rephrase the sentence without resorting to parentheses, ensuring the sentence is grammatically correct while still faithful to the original meaning?

This is a good point. Since we freely remove self-corrections etc, it would be better to not use parentheses for minor corrections and additions which obviously remain true to the original meaning. It could be left to the reviewer to self assess what is appropriate - to include parentheses or a (?) in cases where they are not confident in their editing.